pkg/controller/job/metrics/metrics.go

```diff
@@ -37,21 +40,25 @@ var (
 			StabilityLevel: metrics.ALPHA,
 			Buckets:        metrics.ExponentialBuckets(0.001, 2, 15),
 		},
-		[]string{"completion_mode", "result"},
+		[]string{"completion_mode", "result", "action"},
```

Comment:
add this change to the release note

Reply:
Definitely, it's a breaking metrics change.

Reply:
Not really breaking considering that this metric didn't hit a release yet.